docs/user/02-logs.md

```diff
@@ -476,21 +476,29 @@ The maximum amount of LogPipelines is 5.
 
 ## Troubleshooting
 
-### Logs Not Arriving at the Destination
+### No Logs Arrive at the Backend
 
-   Cause: Wrong backend endpoint configuration or authentication credentials are used.
+Cause: Incorrect backend endpoint configuration (e.g., using the wrong authentication credentials) or the backend being unreachable.
 
-   Remedy: Investigate the cause with the following steps:
-   1. Check the `telemetry-fluent-bit` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`.
+Remedy: 
+- Check the `telemetry-fluent-bit` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`.
+- Check if the backend is up and reachable.
 
-### Influx Capacity Reaching Its Limit
+### Not All Logs Arrive at the Backend
 
-  Cause: The backend ingestion rate is too small for the data influx.
+Cause: The backend is reachable and the connection is properly configured, but some logs are refused.
```

Comment:
That's still a symptom, not a cause.
What's the actual cause, is the buffer filling up?
Not sure if this and the following section should really be split in two, considering the remedy is related?

Is there a common remedy procedure, like ...:

>1. Check the error logs.
>2. If backend is refusing logs and limiting the rate, you have the following options:
>   - Increase ingestion rate (...) in the backend (...)
>   - Decrease ingestion rate (...)
>   - Reduce emitted logs (...)


But what if you check the logs and "If backend is refusing logs limiting the rate" does not apply? Wha's the remedy then?

Reply:
Not sure I'm happy about the options "increase ingestion rate" and "decrease ingestion rate" next to each other; can we phrase them more clearly? I'm not yet sure how. I know you explained and I understand what we're trying to say, but I'm not sure the text is clear enough at this point.